examples/07_advanced/plot_ica_neurovault.py

```diff
@@ -39,13 +39,17 @@
 print("Fetching Neurovault images; "
       "if you haven't downloaded any Neurovault data before "
       "this will take several minutes.")
-nv_data = fetch_neurovault(max_images=80, fetch_neurosynth_words=True)
+nv_data = fetch_neurovault(max_images=30, fetch_neurosynth_words=True)
 
 images = nv_data['images']
 term_weights = nv_data['word_frequencies']
 vocabulary = nv_data['vocabulary']
+if term_weights is None:
+    term_weights = np.ones((len(images), 2))
+    vocabulary = np.asarray(["Neurosynth is down", ":("])
 
 # Clean and report term scores
+print(term_weights)
```

Comment:
Should this be printed to screen or returned ?

Reply:
this is not in a function but in the example. but good point! there is no reason to print it